### ISSUE TITLE
Create and remove keyspaces and columnfamilies

### ISSUE THREAD

**Issue by mateo41** [driftx/Telephus#5]
Good afternoon,

Cassandra 0.7 has a feature to create and remove keyspaces at runtime. I have an application that uses Telephus as a client to our Cassandra datastore. We would like to create and remove keyspaces. 

I've noticed that the unstable branch has this ability to create and remove keyspaces (system_add_keyspace, system_drop_keyspaces). I see there are unittests, but I haven't tried to run them yet. Are these methods working? When do you think these changes will be merged into the master branch? I might be able to provide some of my time to merge these changes into the master branch. 

Thanks,
Matt

**Comment by driftx** [comment]
These calls work, yes.  They won't be merging to master anytime soon, since that is currently for tracking the Cassandra 0.6 branch.  Unstable will merge to the 0.7 branch very shortly and 0.7 will have another release (someone's working on it right now.)  You can use unstable if you need it sooner.

**Comment by thepaul** [comment]
Merged to 0.7 now, as a matter of fact.

**Comment by thepaul** [comment]
You can get source packages and a build, along with thrift, if you add

```
deb http://debian.riptano.com/debian unstable main
deb-src http://debian.riptano.com/debian unstable main
```

to your sources.list.
